### PR TITLE
Modules changes

### PR DESCRIPTION
**Pull request by nirarg**

### CODE REVIEW

File: examples/rosa-classic-public-with-byo-vpc/variables.tf [hunk 1]
```diff
@@ -0,0 +1,12 @@
+variable "openshift_version" {
+  type    = string
+  default = "4.13.13"
```
Comment (bardielle) [inline examples/rosa-classic-public-with-byo-vpc/variables.tf:3]: Maybe we should update the default?

File: modules/vpc/main.tf [hunk 2]
```diff
@@ -14,6 +8,9 @@ resource "aws_vpc" "site" {
     },
     var.tags,
   )
+  lifecycle {
```
Comment (bardielle) [inline modules/vpc/main.tf:11]: There is a jira ticket for tracking it?

**Comment by bardielle** [comment]
/hold
(till @nirarg will fix 2 comments)

File: examples/rosa-classic-public-with-byo-vpc/variables.tf (hunk 1 above)
Comment (nirarg) [reply examples/rosa-classic-public-with-byo-vpc/variables.tf:3]: I will change it in different PR

**Pull request merged**